CharlieBackend.Business/Services/AttachmentService.cs

```diff
@@ -116,5 +106,69 @@ public async Task<Result<DownloadAttachmentDto>> DownloadAttachmentAsync(long at
 
             return Result<DownloadAttachmentDto>.GetSuccess(downloadedAttachment);
         }
+
+        public async Task<Result<AttachmentDto>> DeleteAttachmentAsync(long attachmentId)
+        {
+            var attachment = await _unitOfWork.AttachmentRepository.GetByIdAsync(attachmentId);
+
+            if (attachment == null)
+            {
+                return Result<AttachmentDto>.GetError(ErrorCode.ValidationError,
+                     "Attachement with id: " + attachmentId + " is not found");
+            }
+
+            await _blobService.DeleteAsync(attachment.ContainerName);
+
+            await _unitOfWork.AttachmentRepository.DeleteAsync(attachmentId);
+
+            await _unitOfWork.CommitAsync();
+
+            return Result<AttachmentDto>.GetSuccess(_mapper.Map<AttachmentDto>(attachment));
+        }
+
+        public bool AttachmentsExtentionValidation(IFormFileCollection fileCollection)
+        {
+            string[] dangerousExtentions = 
```

Comment:
I think its settings of project, and better be somewhere at .json settings

Reply:
Its not setting of project